netsuite/client.bal

```diff
@@ -292,10 +302,10 @@ public client class Client {
         return getCustomerResult(response);
     }
 
-    # Gets a contact record from Netsuite by using internal ID
+    # Gets a contact record from Netsuite by using internal ID.
```

Comment:
Change "using internal ID" to "using the internal ID" or "using an internal ID" as relevant.